server/errors.js

```diff
@@ -107,5 +108,14 @@ const errors = Object.freeze({
     8000: 'failed to add feedback',
     8001: 'failed to get feedback',
     8002: 'failed to remove all feedback',
+
+    //9000 virtual file system
+    9000: 'failed to clean the virtual file system',

```

Comment:
You need to add these error codes to the UI common.js

Reply:
fixed